app/core/utils.js

```diff
@@ -1356,10 +1356,12 @@ const arenas = [
   { season: 12, slug: 'supercharged', type: 'championship', start: new Date('2024-12-01T00:00:00.000-08:00'), end: new Date('2025-01-01T00:00:00.000-08:00'), results: new Date('2025-01-14T07:00:00.000-08:00'), levelOriginal: '66f545e57e91e7168c3e463c', tournament: '6756bac52bfcb2c7059f3cb3', image: '/file/db/level/66f545e57e91e7168c3e463c/superchargedbanner2.jpg' },
   { season: 13, slug: 'pawns-passage', type: 'regular', start: new Date('2025-01-01T00:00:00.000-08:00'), end: new Date('2025-06-01T00:00:00.000-07:00'), results: new Date('2025-06-14T07:00:00.000-07:00'), levelOriginal: '675a76867ea2b689e0f86e87', image: '/file/db/level/675a76867ea2b689e0f86e87/PawnsPassageBanner.jpg', tournament: '6810a48b4fb20ed54d4ddc7d' },
   { season: 13, slug: 'kings-gambit', type: 'championship', start: new Date('2025-05-01T00:00:00.000-07:00'), end: new Date('2025-06-01T00:00:00.000-07:00'), results: new Date('2025-06-10T07:00:00.000-07:00'), levelOriginal: '679b1495454eb6d46f27e050', image: '/file/db/level/679b1495454eb6d46f27e050/KingsGambit.jpg', tournament: '6810a617882c6fe46452003d' },
-  { season: 14, slug: 'strikers-stadium', type: 'regular', start: new Date('2025-06-01T00:00:00.000-07:00'), end: new Date('2025-08-01T00:00:00.000-07:00'), results: new Date('2025-09-14T07:00:00.000-07:00'), levelOriginal: '', image: '' },
-  { season: 14, slug: 'golden-goal', type: 'championship', start: new Date('2025-06-01T00:00:00.000-07:00'), end: new Date('2025-08-01T00:00:00.000-07:00'), results: new Date('2025-09-14T07:00:00.000-07:00'), levelOriginal: '', image: '' },
-  { season: 15, slug: 'turbo-track', type: 'regular', start: new Date('2025-08-01T00:00:00.000-08:00'), end: new Date('2026-01-01T00:00:00.000-08:00'), results: new Date('2026-01-10T07:00:00.000-08:00'), levelOriginal: '', image: '' },
-  { season: 15, slug: 'grand-prix', type: 'championship', start: new Date('2025-08-01T00:00:00.000-08:00'), end: new Date('2026-01-01T00:00:00.000-08:00'), results: new Date('2026-01-10T07:00:00.000-08:00'), levelOriginal: '', image: '' },
+  // We have only one arena in this season so just fake
+  { season: 14, slug: 'turbo-track', type: 'regular', start: new Date('2025-06-01T00:00:00.000-08:00'), end: new Date('2025-06-01T01:00:00.000-08:00'), results: new Date('2025-06-10T07:00:00.000-08:00'), levelOriginal: '', image: '' },
+  
+  { season: 14, slug: 'grand-prix', type: 'championship', start: new Date('2025-06-01T00:00:00.000-08:00'), end: new Date('2025-08-01T00:00:00.000-08:00'), results: new Date('2025-08-01T07:00:00.000-08:00'), levelOriginal: '682aef505b4bd67fa522f11d', image: '' },
+  { season: 15, slug: 'strikers-stadium', type: 'regular', start: new Date('2025-08-01T00:00:00.000-07:00'), end: new Date('2025-12-10T00:00:00.000-07:00'), results: new Date('2025-12-20T07:00:00.000-07:00'), levelOriginal: '', image: '' },
```

Comment:
not urgent but are we not continuing the arena to the end of the year and providing results in January? 

Some people use it during Christmas as family activity, plus it's difficult to process the results/awards during the holiday time as a lot of people are on the leave internally